src/point_one/fusion_engine/messages/configuration.h

```diff
@@ -1439,28 +1439,61 @@ struct P1_ALIGNAS(4) TroposphereConfig {
  * @ingroup config_and_ctrl_messages
  */
 struct P1_ALIGNAS(4) LBandConfig {
-  /** The center frequency of the L-band beam (Hz). */
-  float center_frequency_hz = NAN;
+  /**
+   * The center frequency of the L-band beam (Hz). If `NAN`, use the system
+   * default value.
+   */
+  double center_frequency_hz = NAN;
 
   /**
    * The size of the signal acquisition search space (in Hz) around the center
-   * frequency.
+   * frequency. If `NAN`, use the system default value.
    *
    * For example, a value of 6000 will search +/- 3 kHz around the center
    * frequency.
    */
   float search_window_hz = NAN;
 
+  /**
+   * If `true`, use the provider date rate in this configuration. Otherwise, use
+   * device's default value.
```

Comment:
this is a nit, but the comment should mention WHAT the default value is. (repeat for all of these)